docs/UserGuide.md

```diff
@@ -94,112 +142,74 @@ Example usage:
 
-#### Add details to contact: `add [index] /phone`
+#### Add details to contact: `add [index] [fields]`
 
 You can add a contact detail to an existing contact. No new contact will be created.
 
-Format:
-
-* Add phone: `add [index] /phone [phone]`
-* Add email: `add [index] /email [email]`
-* Add link: `add [index] /link [link]`
-* Add graduation time: `add [index] /grad [semester of graduation]`
-* Add course: `add [index] /course [course]`
-* Add specialisation: `add [index] /spec [specialisation]`
-* Assign priority: `add [index] /priority [priority level]`
-* Add tag: `add [index] /tag [tag name]`
+Format: `add [index] [field prefix] [field value] ...`
 
 Parameters:
 
 * `[index]` is the index of the contact in the list.
-* `[phone]` is a valid phone number (should be all numeric characters).
-* `[email]` is a valid email (`@` (at sign) must be present,
-and `.` (period) must be present after `@` (at sign)).
-* `[link]` is a valid URL linking to a contact’s social media page.
-* `[grad]` is a valid graduation date, in the format `AYxxxx-Sy`.
-    * `xxxx` is the 4-digit representation of the 2 calendar years, in the academic year e.g. `2223` for Academic Year 20`22`/20`23`. Academic year must be between AY1970/1971 to AY2069/2070 (inclusive).
-    * `y` is either `1` for Semester 1, or `2` for Semester 2.
-* `[course]` is a course that the contact is taking/has taken.
-    * `[course]` consists of `course name [/start date] [/end date]`, where `/start` and `/end` indicate
-    when the contact started and finished taking the course respectively. A course can have no dates, a start date, or
-    a start and end date.
-* `[spec]` is the specialisation that contact is taking/has taken.
-* `[priority level]` either **high**, **medium** or **low**, or the initial alphabet to represent the respective priority level.
-* `[tag name]` is the name of the tag to associate the contact with
+* `[field prefix]` specifies the corresponding field to add.
+* `[field value]` is the value to add to the field specified by the preceding prefix.
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: To view a comprehensive list of fields and their prefixes, go to the start of the [Features section](#features).
+
+</div>
 
 Example usage:
 
 * `add 1 /phone 91234567`
     ![add phone](images/add-remark/add-phone.png)
-* `add 3 /email nknguyentdn@gmail.com`
-* `add 1 /link https://nknguyenhc.github.io/`
-* `add 3 /grad AY2223-S1`
-* `add 2 /course CS1101S`
-* `add 3 /spec Robotics & AI`
-* `add 1 /priority high`
-* `add 1 /tag data analyst`
+* `add 2 /grad AY2223-S1`
+* `add 3 /link https://nknguyenhc.github.io /email nknguyentdn@gmail.com `
+* `add 1 /priority high /tag data analyst /course CS1101S /spec Robotics & AI`  
 
 [Table of Contents](#table-of-contents)
 
 ### <u>Category 2 - Edit contact details</u>
 
-#### Edit contact detail : `edit [index] [options]`
+#### Edit contact detail: `edit [index] [field] [options]`
```

Comment:
Should be `[field prefix] [field value]` to be consistent with the section above.

Reply:
I intended the command format in the headers to be a summary of the format. It's consistent across all headers that `[field]` means prefix + value, while `[field prefix]` only appears in delete command header as it appears alone.